root/usr/bin/generate-proxy-cert

```diff
@@ -33,3 +33,6 @@ openssl req -x509 \
   -rand /dev/urandom \
   -batch \
   -days $DAYS
+
+chown foreman-proxy:root $DIR/key.pem $DIR/cert.pem
```

Comment:
I would expect it the other way around: root can read-write and foreman-proxy can read
```suggestion
chown root:foreman-proxy $DIR/key.pem $DIR/cert.pem
```

Reply:
Typo, fixed.